docs/conversion_examples_gallery/imaging/hdf5imaging.rst

```diff
@@ -0,0 +1,28 @@
+HDF5 data conversion
+^^^^^^^^^^^^^^^^^^^
+
+Convert HDF5 imaging data to NWB using :py:class:`~nwb_conversion_tools.datainterfaces.ophys.hdf5.hdf5datainterface.Hdf5ImagingInterface`.
+
+.. code-block:: python
+
+    >>> from datetime import datetime
+    >>> from dateutil import tz
+    >>> from pathlib import Path
+    >>> from nwb_conversion_tools import Hdf5ImagingInterface
+    >>>
+    >>> file_path = OPHYS_DATA_PATH / "imaging_datasets" / "hdf5" / "demoMovie.hdf5"
+    >>> interface = Hdf5ImagingInterface(file_path=file_path, verbose=False)
+    >>>
+    >>> metadata = interface.get_metadata()
+    >>> metadata.update(NWBFile=dict())
+    >>> # For data provenance we add the time zone information to the conversion
+    >>> session_start_time = datetime(2020, 1, 1, 12, 30, 0, tzinfo=tz.gettz("US/Pacific"))
+    >>> metadata["NWBFile"].update(session_start_time=session_start_time)
```

Comment:
I think it would be simpler to add the NWBFile
```suggestion
    >>> metadata = interface.get_metadata()
    >>> metadata.update(NWBFile=dict(session_start_time=)datetime(2020, 1, 1, 12, 30, 0, tzinfo=tz.gettz("US/Pacific")))
```

Reply:
I'd support the one-liner Ben suggests.